<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QMainWindow" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1121</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>连连看</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>分数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tipsButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>340</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>460</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>691</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>220</y>
      <width>127</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image:
url(:/button/buttons/button.png)</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>280</y>
      <width>127</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image:
url(:/button/buttons/button.png)</string>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>340</y>
      <width>127</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image:
url(:/button/buttons/button.png)</string>
    </property>
    <property name="text">
     <string>提示: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>460</y>
      <width>127</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image:
url(:/button/buttons/button.png)</string>
    </property>
    <property name="text">
     <string>        退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>460</y>
      <width>127</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华光超粗黑_CNKI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/bg/bg/score.png);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>400</y>
      <width>127</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image:
url(:/button/buttons/button.png)</string>
    </property>
    <property name="text">
     <string>设置/帮助</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="coverLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>651</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bg/bg/board.png);</string>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {   border: 2px solid grey;  border-radius: 5px;   background-color: #FFFFFF;}QProgressBar::chunk {   background-color: #05B8CC;   width: 20px;}QProgressBar {   border: 2px solid grey;  border-radius: 5px;   text-align: center;}
</string>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
    <property name="format">
     <string>%vs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>-120</x>
      <y>-50</y>
      <width>1381</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/bg/bg/bg1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="levelLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>关卡：</string>
    </property>
   </widget>
   <widget class="QLabel" name="levelLabel_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>127</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image:
url(:/button/buttons/button.png)</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>胜利</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipsTImesLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>340</y>
      <width>61</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="autoSolveButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>150</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/button/buttons/button.png)}</string>
    </property>
    <property name="text">
     <string>自动解题</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>widget</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>scoreLabel_2</zorder>
   <zorder>label_7</zorder>
   <zorder>exitButton</zorder>
   <zorder>pauseButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>coverLabel</zorder>
   <zorder>progressBar</zorder>
   <zorder>settingButton</zorder>
   <zorder>scoreLabel</zorder>
   <zorder>levelLabel</zorder>
   <zorder>levelLabel_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>tipsTImesLabel</zorder>
   <zorder>tipsButton</zorder>
   <zorder>autoSolveButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
